Sources/Sentry/SentryHub.m

```diff
@@ -83,14 +83,15 @@ - (void)closeCachedSession {
     SentryFileManager *fileManager = [[self getClient] fileManager];
     SentrySession *session = [fileManager readCurrentSession];
     if (nil != session) {
+        [SentryLog logWithMessage:[NSString stringWithFormat:@"A cached session was found."] andLevel:kSentryLogLevelDebug];
```

Comment:
`l`: What about prefixing all the log messages here with `closeCachedSession` or something similar. The log message itself doesn't tell me this is happening during closing a session.

Reply:
Logging is generally suboptimal, I usually just search for the string if I want to get to that line of code. We could pass a "category" to a logger instance 

>The log message itself doesn't tell me this is happening during closing a session.

The log message is just saying it found a cached file